### DISCUSSION TITLE
Ruby CPU memory usage metrics

### DISCUSSION THREAD

**Discussion by guangzhengli-myob** [open-telemetry/opentelemetry-ruby-contrib#840] [Q&A]
I was wondering if there are currently any methods available within Ruby OpenTelemetry SDK to gather CPU and memory usage metrics specifically for Ruby on Rails applications. 

Additionally, are there any future plans to implement such capabilities?

Thanks!

**Comment by arielvalentin** [comment]
No there isn't a way unfortunately with the Ruby SDK.

The metrics SDK in under development. Once it is GA we will be able to provide the ability to gather core stats.
